proposals/NNNN-sdl-js-pwa.md

```diff
@@ -0,0 +1,213 @@
+# WebEngine support for SDL JavaScript
+
+* Proposal: [SDL-NNNN](NNNN-sdl-js-pwa.md)
+* Author: [Kujtim Shala](https://github.com/kshala-ford)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / RPC / JavaScript / Server / Local ]
```

Comment:
I guess it can be removed because it's not an impacted platform. The proposal is about apps running *locally* on the system but there's no impact as this is something new with this proposal... should I just remove it?

Reply:
Yes, I think removing would be best.  Thanks!